conveyal/analysis-ui
# Ensure regional analysis can be deleted while running

## Description

Deleting a running analysis silently fails. This fixes that by properly passing the analysis id.

## How to test

1. Create a regional analysis. 
2. Delete it while running. Ensure it is deleted.
